components/ai-services-mgt/org.wso2.carbon.ai.service.mgt/pom.xml

```
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Check and change other places as well
```suggestion
  ~ Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```

---

Fixed with 3f1f8db8138db6f024ecb911a91be0c30a0d1ae6